src/_internals/item/Item.unit.tsx

```diff
@@ -41,6 +41,16 @@ describe('Item', () => {
     expect(wrapper.hasClass('kirk-item--highlighted')).toBe(true)
   })
 
+  it("Shouldn't be wrappable by default", () => {
+    const wrapper = shallow(<Item leftTitle="Test" />)
```

Comment:
Even if this test file is using enzyme, I would recommend using RTL for new tests.

```
    render(<Item leftTitle="Test" />)
    expect(screen.getByText('Test')).toHaveClass(kirk-item--wrappable')
```

Reply:
Done